wdio/specs/news-item.js

```diff
@@ -20,4 +20,13 @@ describe('Municipality news item page', () => {
     browser.waitForVisible('div=إدارة مجموعات تعلن: الفرصة الأخيرة لإغلاق ديون ضريبة الأملاك للسكان. عجل تسوية الديون الخاصة بك وتجنب العقوبات. يسري هذا العرض حتى -8.4.17.');
   });
 
+  it('should not show an expired news item', () => {
```

Comment:
I think this is more a test for the backend, but if time doesn't permit we can skip it/ add in the future